packages/coinstac-ui/app/render/state/ducks/remote-results.js

```diff
@@ -20,6 +20,24 @@ export const fetch = applyAsyncLoading(id => (dispatch) => {
     });
 });
 
+export const fetchConsortiaResults = applyAsyncLoading(() => {
+  const dbRegistry = app.core.dbRegistry;
+
+  return (dispatch) => {
+    return app.core.consortia
+    .then((docs) =>
+      Promise.all(docs.map(({ _id }) =>
+        dbRegistry.get(`remote-consortium-${_id}`)
+      ))
+    )
+    .then((responses) => {
+      const results = sortBy(flatten(responses), ['endDate', 'startDate']).reverse();
+      dispatch(setRemoteResults(results));
+      return results;
+    });
+  };
+});
+
```

Comment:
Perhaps there needs to be a separate action for results from all consortia versus results from a single consortium?